### ISSUE TITLE
Villagers and Ticket Barriers

### ISSUE THREAD

**Issue by ProDeveloperAlex** [jonafanho/Minecraft-Transit-Railway#307]
I love this mod. But, somethings **wrong.**
Villagers and Ticket Barriers... are not the best combo.

What I'm proposing...
Make villagers able to walk in and out of ticket barriers.
They get stuck in them, a lot, so this would be a step up for realistic features.

**Comment by Minecraftispro** [comment]
> What if the villagers rode the trains too?
i forgot to put that in too

**Comment by Ultimate-USA-YT** [comment]
Trains + Villagers = Commuters

**Comment by TheBreadGuy102** [comment]
it won't be great for stations/railways without APGs/PSDs. if added, it should be configurable in the settings

**Comment by minecraftLYC** [comment]
Yes! I want villagers can take train because when only one player in the train ,it is too alone.

**Comment by Leoparking** [comment]
对的，，我更希望所有动物都能被吸上车，就像原版的矿车一样

**Comment by FilmerBoi** [comment]
Would this be able to work with Minecraft Comes Alive Reborn? If it can, it would make my railways and villages connected by them so much livelier.
